src/app/Services/order.service.ts

```diff
@@ -10,13 +10,19 @@ export class OrderService implements CRUD<any> {
   url = 'http://localhost:5298/api/Order';
   constructor(private http: HttpClient) {}
   getById(id: number): Observable<any> {
-    return this.http.get(`${this.url}/GetOrderById${id}`);
+    return this.http.get(`${this.url}/GetOrderById/${id}`);
   }
   create(data: Order): Observable<any> {
     return this.http.post(`${this.url}`, data);
   }
-  update(id: number, data: Order): Observable<any> {
-    return this.http.put(`${this.url}/${id}`, data);
+  createByAdmin(data: Order): Observable<any> {
```

Comment:
Edit the create function to check if the user is `admin or Employee`  or `Seller` and call the proper API.